src/account/BiconomySmartAccountV2.ts

```diff
@@ -978,7 +979,7 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
         return this.getPaymasterAndData(partialUserOp, {
           ...paymasterServiceData,
           feeTokenAddress: feeQuote.tokenAddress,
-          calculateGasLimits: true // Always recommended and especially when using token paymaster
+          calculateGasLimits: paymasterServiceData.calculateGasLimits ?? true // Always recommended and especially when using token paymaster
```

Comment:
so they will manually pass false in order to use gas offsets?
will this apply for every use case tx paths?

Reply:
calculateGasLimits are set to false in the SDK depending if gasOffsets are use or not

`userOp = await this.getPaymasterUserOp(userOp, {
          ...buildUseropDto.paymasterServiceData,
          calculateGasLimits: false
        })`